### DISCUSSION TITLE
Wrong numbering in bibliography

### DISCUSSION THREAD

**Discussion by Quelhas00** [joaomlourenco/novathesis#448] [Q&A]
Hello, I am doing my master thesis at NOVA SST and I am using version [7.0.3] in overleaf.

I was using the previous version [6.10.10] and the bibliography was fine. When I upgraded to the newest version [7.0.3] the bibliography stopped working right, the numeration is always zero. I did not change any of the code from the previous version.

![Captura de ecrã 2023-07-11 105541](https://github.com/joaomlourenco/novathesis/assets/139223591/32a82280-a1b6-433b-aa04-dae4004d2f74)

Is there anything I can do?

**Comment by joaomlourenco** [comment]
That happens when you have problems with the bibliography.  Please open the file `template.log` and search for "error" or "Error".  If you find nothing, then search for "biber" and see carefully any error or warning messages.  

Something is wrong with your bibliography and biblatex/biber is aborting and not numbering the references right.
